imperial_coldfront_plugin/views.py

```diff
@@ -1 +1,53 @@
 """Plugin views."""
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.decorators import login_required
+from django.http import HttpRequest, HttpResponse, HttpResponseForbidden
+from django.shortcuts import get_object_or_404, render
+
+from .models import GroupMembership
+
+User = get_user_model()
```

Comment:
This feels wrong... Shouldn't we be using `get_user_model()` directly each time?

Reply:
By assigning `get_user_model()` to `User`, we establish a consistent reference to the user model throughout the module. This is useful if our module is extended or reused within the project, as we see in line 43. Assigning it to `User` at the module level ensures that the function is called only once and reduces redundant lookups.

Also, if multiple parts of the codebase refer to `User`, maintaining a single reference (`User`) is easier to manage than using `get_user_model()` everywhere.

Moreover, I think that in project-specific code (non-reusable apps), it is common and entirely safe to `assign get_user_model()` to `User` because the user model (`AUTH_USER_MODEL`) is stable and does not change dynamically during runtime.